src/ansys/fluent/core/session_shared.py

```diff
@@ -67,9 +69,10 @@ def _make_datamodel_module(session, module_name):
 
         file_name = datamodel_file_name_map[module_name]
         module = pyfluent.utils.load_module(
-            f"{module_name}_{session._version}",
+            f"{module_name if module_name != 'MeshingUtilities' else file_name}_{session._version}",
```

Comment:
@prmukherj 
- This kind of mysterious edge case handling deserves a comment
- The special handling is for one value of one of the arguments. Therefore the caller should handle the problem. `_make_datamodel_module()` can accept an additional argument `module_file_name=None`